### ISSUE TITLE
[A11Y] Use `aria-live` on changing regions (discussion lists, post streams)

### ISSUE THREAD

**Issue by davwheat** [flarum/framework#3365]

**Comment by davwheat** [comment]
This isn't actually needed. Looking more in-depth, I've found that `aria-live` should really only be used when focus should be thrown away from the current task onto the changed region(s).
